### PR TITLE
using helm  install aeraki

### PR DESCRIPTION
**Pull request by ClareChu**
```bash
helm install aeraki ./aeraki -n istio-system
```

### CODE REVIEW

**Comment by ClareChu** [comment]
@zhaohuabing

File: manifests/charts/aeraki/values.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  create: false
+  type: ClusterIP
+  port: 80
+
+ingress:
```
Comment (zhaohuabing) [inline manifests/charts/aeraki/values.yaml:49]: Do we need an ingress?

File: manifests/charts/aeraki/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{{- if .Values.ingress.enabled -}}
```
Comment (zhaohuabing) [inline manifests/charts/aeraki/templates/ingress.yaml:1]: Do we need an Ingress?

Comment (ClareChu) [reply manifests/charts/aeraki/templates/ingress.yaml:1]: No, so I set it to off

File: manifests/charts/aeraki/values.yaml (hunk 1 above)
Comment (ClareChu) [reply manifests/charts/aeraki/values.yaml:49]: But I'm not sure if there's a need for this in the future, so I'll save it for now, and if it's not necessary I can delete it

File: manifests/charts/aeraki/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: aeraki
+description: A Helm chart for Kubernetes
```
Comment (zhaohuabing) [inline manifests/charts/aeraki/Chart.yaml:3]: Could we modify the default description to the introduction of Aeraki?

Comment (ClareChu) [reply manifests/charts/aeraki/Chart.yaml:3]: ok

File: manifests/charts/aeraki/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: aeraki
+description: Manage any layer 7 traffic in Istio service mesh!
+
+# A chart can be either an 'application' or a 'library' chart.
```
Comment (zhaohuabing) [inline manifests/charts/aeraki/Chart.yaml:5]: I suggest we add some introduction here so the user knows how to properly install the required Istio component.

For example:
1. Require Istio 10.0.1 or above
2. Require smart dns enabled
See https://github.com/aeraki-framework/aeraki/blob/master/test/e2e/common/istio-config.yaml for Istio installation configuration

**Review by zhaohuabing — CHANGES_REQUESTED** [review]

File: manifests/charts/aeraki/Chart.yaml (hunk 4 above)
Comment (ClareChu) [reply manifests/charts/aeraki/Chart.yaml:5]: smart dns, use helm to manage more appropriate

Comment (ClareChu) [reply manifests/charts/aeraki/Chart.yaml:9]: the description  fields Just the introduction function, I think it is more appropriate to explain in the readme.md

Comment (ClareChu) [reply manifests/charts/aeraki/Chart.yaml:9]: I think it is necessary to add a copy of how to install aeraki with istio installed,The same question has been raised

**Review by zhaohuabing — APPROVED** [review]
LGTM

**Pull request merged**